src/frontend/src/pages/Projects/ProjectsDataTable.jsx

```diff
@@ -24,18 +26,26 @@ export function ProjectsDataTable({ projects }) {
         { key: 'ownerNames', header: 'Owner(s)' }
     ];
 
+    const navigate = useNavigate();
+
     const getRow = rowId => projects.find(({ id }) => id == rowId);
 
     return (
         <SmsDataTable rows={projects} headers={headers} className='projects-page__datatable'>
             {({ rows, headers, getHeaderProps, getRowProps, getTableProps, onInputChange }) => (
                 <TableContainer
                     title='Projects'
-                    description='This is a list of all the project for this semsester.'
+                    description='This is a list of all the project for this semester.'
                 >
                     <TableToolbar>
                         <TableToolbarContent>
                             <TableToolbarSearch defaultExpanded={true} onChange={onInputChange} />
+                            <Button 
+                                onClick={() => navigate(`${AppRoutes.projects}/add`)}
+                                size="small" 
+                                kind="primary"> 
+                                Create a new project
+                            </Button>
```

Comment:
likewise as above, however projects won't be added from a regular view. Admins will create projects from their dashboard

```suggestion
```